File: www/FiveStarsPlugin.js
var exec = require('cordova/exec');
var FiveStarsPlugin = {
	overlayAppStore : function (arg0, success, error) {
    exec(success, error, 'FiveStarsPlugin', 'overlayAppStore', [arg0]);
	}	
}




cordova.addConstructor(function () {
  if (!window.plugins) {
    window.plugins = {};
  }

  window.cordova.plugins.FiveStarsPlugin = FiveStarsPlugin;
  return window.cordova.plugins.FiveStarsPlugin;
});

File: src/ios/FiveStarsPlugin.swift
import StoreKit
@objc(FiveStarsPlugin) class FiveStarsPlugin : CDVPlugin {
    @objc(overlayAppStore:)
    func overlayAppStore(command: CDVInvokedUrlCommand) {

        // Error de plugin
        var pluginResult = CDVPluginResult(
        status: CDVCommandStatus_ERROR
        )

        if #available(iOS 10.3,*){
            SKStoreReviewController.requestReview()
            // Success de plugin
            pluginResult = CDVPluginResult(
                status: CDVCommandStatus_OK,
                messageAs: "Ok"
            )
            
        }
        // Enviar respuesta de plugin
        self.commandDelegate!.send(
        pluginResult,
        callbackId: command.callbackId
        )
    }
}
